### PR TITLE
Hopefully fixing revert issue

### PR DESCRIPTION
**Pull request by SancSalix**

### CODE REVIEW

File: post.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```
Comment (cblant20) [inline post.html:1]: you cheated there by including the info under the map with your map image! I wanted you to style that too :P

File: post.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<html>
+	<head>
+  <link type ="text/css" rel="stylesheet" href="style.css">
```
Comment (cblant20) [inline post.html:4]: what's up with this indentation?

File: post.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	</head>
+    
+	<body>
+    <header> </header>
```
Comment (cblant20) [inline post.html:8]: empty header?

File: post.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    
+    <div class="main">
+    <ul class ="disclaimer">
```
Comment (cblant20) [inline post.html:11]: make sure to visually nest everything that is nested in actuality

File: post.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      </li>
+
+      <li>
+        <form>       
+          <input type ="radio" id="prohibit">
+          <label for="prohibit"><a href="https://www.craigslist.org/about/prohibited" target="_blank"><u>prohibited</u></a></label>
+        </form>
+      </li>
+
+      <li><p>Posted <sub><u>3 days ago</u></sub></p></li>
```
Comment (cblant20) [inline post.html:25]: <u> is outdated.

Comment (cblant20) [reply post.html:25]: < u >

File: style.css [hunk 6]
```diff
@@ -0,0 +1,120 @@
+body {
+  background-color: white;
+  font-family: "Times New Roman", sans-serif;
+}
+
+html, body, h1, h2, h3, p, ol, ul, li, a, form, label {
+padding: 0;
```
Comment (cblant20) [inline style.css:7]: tabbing here too!

File: style.css [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+.main {
+    margin: 0 100px 0 250px;
+ }
+
+.disclaimer {
+	margin: -10px 0 0 0;
+}
+
+.disclaimer li {
+	display: inline-flex;
```
Comment (cblant20) [inline style.css:29]: why so many indentations?!

**Comment by SancSalix** [comment]
Closing so I don't see this anymore haha

**Pull request closed without merging**